src/database/db_data.sql

```diff
@@ -1,23 +1,17 @@
 BEGIN;
 
-INSERT INTO currencies (id, name, code) VALUES
-(1, 'US Dollar', 'USD'),
-(2, 'Euro', 'EUR'),
-(3, 'Bitcoin', 'BTC'),
-(4, 'Ethereum', 'ETH');
+INSERT INTO users (username, email, password) VALUES ('Matteo', 'matteo@tic.it', '$2a$10$/VEP0V7yxcHvnen09h1VHuJ73wlK1wSSuFOKzC.vpBdwePR2zxl96');
+INSERT INTO users (username, email, password) VALUES ('Iannis', 'iannis@tic.it', '$2a$10$un8zH8z.2UOEUAA..q0HNej7Zg0OtZ1HJVXhMIE0TdOdP05jAhceK');
+INSERT INTO users (username, email, password) VALUES ('Alberto', 'alberto@tic.it', '$2a$10$cO4d/xEHBilynVXB3eWV7ejQoe13skYhV5MQmkLrGYsnM1YHw/QWO');
+INSERT INTO users (username, email, password) VALUES ('Claudio', 'claudio@tic.it', '$2a$10$c2Uszq3mNMMpWtoSe.ehpObRkwD/xkdtb9bQv2F6iqm2pqrJmMA0K');
+INSERT INTO users (username, email, password) VALUES ('Giulia', 'giulia@tic.it', '$2a$10$pgzgeR9rC55BIlLdF/Y.xOESABUA3qhtfy7YPCgyxpgM2KwQwijgq');
 
-INSERT INTO rates (fromcurrency_id, tocurrency_id, rate, timestamp) VALUES
-(1, 2, 1.2, current_timestamp),
-(1, 3, 1.2, current_timestamp),
-(1, 4, 1.2, current_timestamp),
-(2, 1, 1.5, current_timestamp),
-(2, 3, 1.5, current_timestamp),
-(2, 4, 1.5, current_timestamp),
-(3, 1, 1.5, current_timestamp),
-(3, 2, 1.2, current_timestamp),
-(3, 4, 1.2, current_timestamp),
-(4, 1, 0.7, current_timestamp),
-(4, 2, 0.7, current_timestamp),
-(4, 3, 0.7, current_timestamp);
+
+INSERT INTO bookmarks (user_idd, already_known, interested_in, starred) VALUES
+(1, '{"id": "2,5,8"}','{"id": "2,5,8"}','{"id": "2,5,8"}'),
```

Comment:
why instead of `'{"id": "2,5,8"}'` you are not using `'[2,5,8]'`?

Reply:
modified id names